app/main.py

```diff
@@ -15,3 +15,10 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :return: count occurrences of letter in phrase
     """
     # write your code here
```

Comment:
Please, remove this comment, when you implement your solution

Reply:
This line is still here, try to make new commit and new push.
Take a look into the Guide.